logic/peers.go

```diff
@@ -122,7 +122,7 @@ func GetPeerUpdateForHost(network string, host *models.Host, allNodes []models.N
 			}
 			peerConfig := wgtypes.PeerConfig{
 				PublicKey:                   peerHost.PublicKey,
-				PersistentKeepaliveInterval: &peer.PersistentKeepalive,
+				PersistentKeepaliveInterval: &peerHost.PersistentKeepalive, // TODO remove?
 				ReplaceAllowedIPs:           true,
```

Comment:
remove the TODO comment, it's necessary to set PKA  there

Reply:
https://github.com/gravitl/netmaker/pull/2604/commits/84ccc5828f17ba740f53c567158f4f089f588db8